src/bartholomew.rs

```diff
@@ -114,10 +143,32 @@ pub fn render(req: Request) -> Result<Response> {
                         .clone()
                         .unwrap_or_else(|| DEFAULT_CONTENT_TYPE.to_owned());
 
+                    let disable_page_cache = doc.head.disable_page_cache;
+
                     let data = engine
                         .render_template(doc, config, req.headers().to_owned())
                         .map_err(|e| anyhow!("Rendering {:?}: {}", &content_path, e))?;
 
+                    if !disable_cache {
+                        match disable_page_cache {
```

Comment:
I think `if !disable_page_cache.unwrap_or_default()` would let you avoid the empty `Some(true)` case and the extra layer of nesting it inflicts on you.

Reply:
Thanks for pointing that out!